source/js/utils/computeMathOperations.js

```diff
@@ -1,13 +1,17 @@
+function fixDecimal(a) {
+  return (parseFloat(number).toPrecision(12));
```

Comment:
It throws an error:
`Uncaught ReferenceError: number is not defined`

You need to change `a` to `number` and it will work. :)

Reply:
obviously... :)